app/javascript/components/maps/map/index.js

```diff
@@ -203,9 +203,15 @@ class MapContainer extends PureComponent {
   }, 300);
 
   handleMapInteraction = e => {
-    const { draw, menuSection } = this.props;
+    const { draw, menuSection, selectedInteraction } = this.props;
     if (!draw && !menuSection && e.features && e.features.length) {
       this.props.setInteraction(e);
+      track('mapInteraction', {
+        label:
+          selectedInteraction &&
```

Comment:
This will be the selected interaction of the previous click. All the current event data is inside e. Update to reflect.